docs/content/usage-reporting.md

```diff
@@ -117,6 +117,8 @@ Download and save the deployment file [cluster-connector.yaml](https://raw.githu
 - `-nms-server-address` should be the address of the Usage Reporting API, which will be the combination of NGINX Management Suite server hostname and the URI `api/platform/v1`
 - `nms-basic-auth-secret` should be the namespace/name of the secret created in step 3: `nginx-cluster-connector/nms-basic-auth`.
 
+{{< note >}} When using RedHat OpenShift platform, create SecurityContextConstraints object located in `shared-examples/` directory as a prerequisite for Nginx Cluster Connector using `oc create -f scc.yaml` {{< /note >}}
```

Comment:
```suggestion
{{< note >}}  OpenShift requires a SecurityContextConstraints object for NGINX Cluster Connector. 

It can be created with the command `oc create -f scc.yaml`, using the example file found in `[shared-examples/usage_reporting](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/shared-examples/usage-reporting)` {{< /note >}}
```

Reply:
@ADubhlaoich I'll would rather point it to a release tag instead of main branch so will update it post release, will update the rest